app/src/main/java/com/teamltt/carcare/activity/HomeActivity.java

```diff
@@ -146,27 +147,6 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
-    protected void goToDemo(View view) {
-        Intent intent = new Intent(this, DemoActivity.class);
-        startActivity(intent);
-    }
-    protected void goToStatic(View view) {
-        Intent intent = new Intent(this, DemoActivity.class);
-        startActivity(intent);
-    }
-    protected void goToDynamic(View view) {
-        Intent intent = new Intent(this, DemoActivity.class);
-        startActivity(intent);
-    }
-
-    /*protected void openDrawer(View view) {
-        if (drawer.isDrawerOpen(findViewById(android.R.id.home))) {
-            drawer.closeDrawer(Gravity.LEFT);
-        }
-        else {
-            drawer.openDrawer(Gravity.RIGHT);
-        }
-    }*/
     public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
I added override annotation since it wasn't there.
The super implementation is used for the return value. It's supposed to handle up navigation I think, but we don't have that so it will be something we could come back to.